src/node-red.njk

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
should we create a directory `src/product` and move the current product page to `index.njk`, allowing this page to fall under `product/node-red`?

Reply:
I'm hesitant as it implies Node-RED is a product of ours?